src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -31,7 +30,7 @@ class OgSelection extends DefaultSelection {
    * @return AccountInterface
    */
   public function getAccount() {
-    if (empty($this->account)) {
+    if (empty($this->currentUser)) {
```

Comment:
I think I may be changing that in the other PR already. So yeah, leaving
here is ok.
On 15 Nov 2015 18:09, "Amitai Burstein" <notifications@github.com> wrote:

> In src/Plugin/EntityReferenceSelection/OgSelection.php
> <https://github.com/amitaibu/og/pull/11#discussion_r44875259>:
>
> > @@ -31,7 +30,7 @@ class OgSelection extends DefaultSelection {
> >     * @return AccountInterface
> >     */
> >    public function getAccount() {
> > -    if (empty($this->account)) {
> > +    if (empty($this->currentUser)) {
>
> @damiankloip <https://github.com/damiankloip> pointed out we could maybe
> just mock the current user, instead of having this method. Anyway, not a
> must for this PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/11/files#r44875259>.
>